### PR TITLE
{phys}[intel/2021a] DFTB+ v21.1 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/c798767c7d5a7ea3dd62136a861f9b96 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/3b17fb82c96a5d97d2c9786d5eb357eb for a full test report.

File: easybuild/easyconfigs/d/DFTB+/DFTB+-21.1_use_dftd3_from_EB.patch [hunk 1]
```diff
@@ -0,0 +1,9 @@
+diff -ru dftbplus-21.1.orig/external/dftd3/CMakeLists.txt dftbplus-21.1/external/dftd3/CMakeLists.txt
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-21.1_use_dftd3_from_EB.patch:1]: missing patch description + author?

File: easybuild/easyconfigs/d/DFTB+/DFTB+-21.1-intel-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['bin/' + x for x in ['dftb+', 'dp_bands', 'dp_dos', 'gen2cif', 'gen2xyz', 'makecube',
+                                   'modes', 'repeatgen', 'straingen', 'waveplot', 'xyz2gen']],
+    'dirs': ['lib/python%(pyshortver)s/site-packages']
+}
+
+sanity_check_commands = [('python', '-c "import dptools"')]
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-21.1-intel-2021a.eb:79]:
```suggestion
sanity_check_commands = ["python -c 'import dptools'"]

File: easybuild/easyconfigs/d/DFTB+/DFTB+-21.1-intel-2021a.eb [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+]
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('dftd3-lib', '0.9'),
+    ('ELSI', '2.7.1', '-PEXSI'),
+]
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/d/DFTB+/DFTB+-21.1-intel-2021a.eb:65]: please drop duplicate empty line

File: easybuild/easyconfigs/d/dftd3-lib/dftd3-lib-0.9-intel-compilers-2021.2.0.eb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+files_to_copy = [
+    (['prg/dftd3', 'test/testapi'], 'bin'),
+    (['lib/libdftd3.a'], 'lib'),
+    (['lib/*.mod', 'prg/*.mod'], 'include'),
+    (['doc/man.pdf', 'CHANGELOG.rst', 'LICENSE', 'README.rst'], 'share'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/dftd3', 'bin/testapi', 'lib/libdftd3.a'],
```
Comment (boegel) [inline easybuild/easyconfigs/d/dftd3-lib/dftd3-lib-0.9-intel-compilers-2021.2.0.eb:35]: Can we add a sanity check command here too?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
b-an02.hpc2n.umu.se - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 3.8.10
See https://gist.github.com/9ca391d05300be27c16142ba5ac3a525 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/14a8820f7fd6ad324dc934cbbaf22a2a for a full test report.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
hp - Linux Debian GNU/Linux 11, x86_64, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz (icelake), Python 3.9.2
See https://gist.github.com/f984e3e49bbb5260b08c51a3a4badd64 for a full test report.

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node2617.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/842e81a88a658f971210461830bacf50 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/b75fc8d3d597ad8de5da2955094ae62f for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Comment by branfosj** [comment]
Going in, thanks @akesandgren!

**Pull request merged**